src/lib/description-helpers/prompts.ts

```diff
@@ -0,0 +1,52 @@
+export const endpointSystemPrompt = `You are an expert, articulate programmer. 
+In 1 sentence and a maximum of 20 words explain how the API endpoint is used in the context of the endpoint based on the example usage of query parameters, request body, and response body provided.
+RULES: 
+1) Focus on useful information for a programmer. 
+2) Be concise. Use grammatically incorrect sentences if it conveys the information more concisely. 
+3) Never repeat the API method, parameter names, parameter types or the website the API is hosted on`;
```

Comment:
Still getting quite not-useful descriptions pretty often with the new prompt:
> Retrieves a list of data, limited by a specified amount, with a JSON response containing details about each retrieved item.
![Screenshot from 2024-05-09 10-23-29](https://github.com/Superflows-AI/openapi-llm/assets/33871096/40785dbc-cf60-4704-a12d-cf8941d327ea)

Reply:
It's not just a one-off:
> Retrieves a list of data objects without requiring any request inputs and it returns a JSON formatted list.

Reply:
My hunch is that the data in the example usage of the endpoint isn't being formatted properly, hence it doesn't have the useful information required

You could give an example input-output pair in the chat history to prime it to give useful answers?

Reply:
Think this is resolved by the temperature change. Would be great to get another test though

Reply:
Wouldn't say it's fixed

![Screenshot from 2024-05-13 11-38-00](https://github.com/Superflows-AI/openapi-llm/assets/33871096/d38de7a6-99ee-444b-8168-04772ffb0048)

Reply:
Fixed it